src/molecules/formfields/SelectField/index.js

```diff
@@ -27,9 +27,25 @@ const renderAdditionalInfo = (func) => {
 };
 
 class SelectField extends React.Component {
-  render() {
+  getClasses = () => {
```

Comment:
what rule caused this refactor?

Reply:
`complexity` - I had to refactor some of the code out of the `render` function because it was too complex

Reply:
file reverted back to master and complexity ignored